src/asmcnc/production/z_head_qc_jig/z_head_qc_home.py

```diff
@@ -86,9 +86,22 @@ def __init__(self, **kwargs):
     def on_enter(self):
         self.update_usb_button_label()
 
+    def get_fw_filepath(self):
+        if int(self.m.s.hw_version) >= 34:
+            return "/media/usb/GRBL*5.hex"
+
+        elif int(self.m.s.hw_version) >= 20:
+            if glob.glob("/media/usb/GRBL*4.hex"):
+                return "/media/usb/GRBL*4.hex"
+
+            # Allow for older FW versions, starting in 2 but not ending in 4
+            return "/media/usb/GRBL23*.hex"
+
+        return "/media/usb/GRBL*.hex"
```

Comment:
If this is GRBL1*, this will mean that multiple files can exist on same USB I think, so may as well :)